ledger-core/v2/insolar/payload/requestcontent_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package payload
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStateRequestContentFlags(t *testing.T) {
+	flags := StateRequestContentFlags(0)
+
+	flags.Set(RequestLatestValidatedCode)
+
+	assert.Equal(t, RequestLatestValidatedCode, flags)
+
+	assert.True(t, flags.Contains(RequestLatestValidatedCode))
+
+	assert.False(t, flags.Contains(RequestImmutableQueue))
+
+	flags.Set(RequestImmutableQueue)
+
+	assert.Equal(t, RequestLatestValidatedCode|RequestImmutableQueue, flags)
+
+	assert.True(t, flags.Contains(RequestLatestValidatedCode))
+
+	assert.True(t, flags.Contains(RequestImmutableQueue))
```

Comment:
If you add test for flags.Set with multiple params it will be good

Reply:
added